cli/flox-rust-sdk/src/providers/build.rs

```diff
@@ -476,7 +476,7 @@ mod tests {
             r#"
        > ❌ ERROR: Build command did not copy outputs to '$out'.
        > - copy a single file with 'cp bin $out'
```

Comment:
Can we fix this while we're in the neighborhood?

```suggestion
       > - copy a single file with 'mkdir -p $out/bin && cp file $out/bin'
       > - copy a bin directory with 'mkdir $out && cp -r bin $out'
```